portfolio/src/main/webapp/style/format.css

```diff
@@ -393,3 +397,57 @@ a {
 .clear {
   clear: both;
 }
+
+#map-container{
```

Comment:
let's also run the formatter through this

Reply:
done